extensions/wikia/FandomCreatorEmail/Controller/ContentUpdatedController.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace FandomCreatorEmail\Controller;
+
+use FandomCreatorEmail\FandomCreatorEmailController;
+
+class ContentUpdatedController extends FandomCreatorEmailController {
+	/** @var string */
+	private $contentTitle;
+
+	/** @var string */
+	private $contentUrl;
+
+	public function initEmail() {
+		parent::initEmail();
+		$this->contentTitle = $this->request->getVal( 'contentTitle' );
+		$this->contentUrl = $this->request->getVal( 'contentUrl' );
+	}
+
+	public function getSubject(): string {
+		return $this->magicWordWrapper->wrap( function() {
+			return $this->getMessage(
+					'emailext-watchedpage-article-edited-subject',
+					$this->contentTitle,
+					$this->getCurrentUserName()
+			)->text();
+		} );
+	}
+
+	/**
+	 * @template avatarLayout
+	 *
+	 * other possible keys:
+	 * 	details - additional email text
+	 * 	editorProfilePage - link to user's profile page on this community
+	 * 	buttonText - button beneath edit summary
+	 * 	buttonLink - where buttonText goes
+	 */
+	public function body() {
+		$this->magicWordWrapper->wrap( function() {
+			$this->response->setData( [
+					'salutation' => $this->getSalutation(),
+					'summary' => $this->getSummary(),
+					'editorUserName' => $this->getCurrentUserName(),
+					'editorAvatarURL' => $this->getCurrentAvatarURL(),
+					'contentFooterMessages' => $this->getContentFooterMessages(),
+					'hasContentFooterMessages' => true,
+			] );
+		} );
+	}
+
+	private function getSummary(): string {
+		return $this->getMessage(
+				'emailext-watchedpage-article-edited',
+				$this->contentUrl,
+				$this->contentTitle
+		)->parse();
+	}
+
+	private function getContentFooterMessages(): array {
+		return [
+				$this->getMessage(
+						'emailext-watchedpage-article-link-text',
+						$this->contentUrl,
+						$this->contentTitle
+				)->parse(),
+		];
+	}
+}
```

Comment:
If you want this email to be able to be sent via `Special:SendEmail`, you'll want to create a `getEmailSpecificFormFields()` method. That method defines the input fields which will be displayed to send that email, and correspond to the arguments that are sent to that emails controller. Eg, DiscussionController defines [this getEmailSpecificFormFields](https://github.com/Wikia/app/blob/dev/extensions/wikia/Email/Controller/DiscussionController.class.php#L83-L109) method, and those fields are the same as the controller is accepting [here](https://github.com/Wikia/app/blob/dev/extensions/wikia/Email/Controller/DiscussionController.class.php#L15-L21).